skt/kerneltree.py

```diff
@@ -248,6 +248,7 @@ def __get_remote_url(self, remote):
         Returns:
             URL of the remote, None if the remote doesn't exist.
         """
+        url = None
```

Comment:
This is a revert of commit cce8dc23. Why complicate the code to achieve the same functionality?

Reply:
The `pylint` complains about it. No return value if `for` loop does not execute, or no match in `if` condition

Reply:
I see it look the same logic here: https://github.com/RH-FMK/skt/blob/master/skt/runner.py#L383

Reply:
What pylint version / configuration are you using? I have seen no such complaints when running it. The code you are linking to should be simplified too, but we can't do everything at the same time :)

Reply:
@veruu my pylint version `1.9.2` :). What about you?

Reply:
Thanks for the help, this warning seems like a newer addition. Okay, let's see how to resolve it without complicating the code too much. What about adding `return None` to the end of the method?

Reply:
That sounds good!

Reply:
For `runner.py`. It has raise exception if not jobid

Reply:
@danghai can you please also clean up the commit message? "Return None if no URL exists", with a mention of the pylint warning it resolves should be enough